### DISCUSSION TITLE
followers_you_know interface

### DISCUSSION THREAD

**Discussion by wuyi771** [tweepy/tweepy#1810] [Questions] [answered]
How should I get the specified account
/followers_you_know, can I only query them one by one? I would like to ask if there is any interface to query /followers_you_know directly

**Comment by Harmon758** [comment, marked as answer]
Unfortunately, there is no Twitter API endpoint for this. You should request it through the [Twitter Developers Forums](https://twittercommunity.com/) or [Twitter Developer Feedback site](https://twitterdevfeedback.uservoice.com/forums/930250-twitter-api) instead.

Currently, the best way is probably to retrieve the user's followers and who the authenticated user is following and check for the intersection.
